terraform/executor/Dockerfile-0.12-slim

```diff
@@ -48,9 +48,10 @@ RUN mkdir -p $TF_PLUGIN_CACHE_DIR
 
 RUN mkdir -p /root/.terraform.d/plugins \
  && for TF_AIVEN_VERSION in $TF_AIVEN_VERSIONS; do \
-      curl -f -L "https://github.com/aiven/terraform-provider-aiven/releases/download/v${TF_AIVEN_VERSION}/terraform-provider-aiven-linux-amd64_v${TF_AIVEN_VERSION}" \
-        -o /root/.terraform.d/plugins/terraform-provider-aiven_v${TF_AIVEN_VERSION} \
-      && chmod +x /root/.terraform.d/plugins/terraform-provider-aiven_v${TF_AIVEN_VERSION}; \
+      curl -f -L "https://github.com/aiven/terraform-provider-aiven/releases/download/v${TF_AIVEN_VERSION}/terraform-provider-aiven_{TF_AIVEN_VERSION}_linux_amd64.zip" -o aiven-provider.zip \
```

Comment:
I believe this is missing a `$` from the 2nd occurrence of `TF_AIVEN_VERSION`

Reply:
Bah. Thank you Chris.